external/segger_rtt/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+FetchContent_Declare(
+    segger_rtt
+    GIT_REPOSITORY https://github.com/SEGGERMicro/RTT
+    GIT_TAG ff57c3d516ea59b0a2d4d5cafb33a48572b09c0b # v7.54
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND ""
+)
+
+FetchContent_MakeAvailable(segger_rtt)
+
+add_library(segger_rtt.rtt ${EMIL_EXCLUDE_FROM_ALL} STATIC)
```

Comment:
I'm looking at the double mention of "rtt". Is it really rtt that is being delivered by the segger_rtt package? The source files provide a stream on rtt, and a tracer on that stream. How about segger_rtt.tracer?

Reply:
Yeah, that annoyed me as well. I don't know if tracer covers it though. segger_rtt.communication? The library facilitates both up- and down channels.

Reply:
Or just segger.rtt?